crates/turborepo-lib/src/run/mod.rs

```diff
@@ -164,6 +167,22 @@ impl Run {
         )
         .build()?;
 
+        if opts.run_opts.graph_file.is_some() || opts.run_opts.graph_dot {
```

Comment:
We should probably just reduce this down to a single field with an enum like `GraphOutput::Stdout, GraphOutput::File(PathBuf)`